content/blog/check_my_repo.md

```diff
@@ -0,0 +1,150 @@
+
+#### Ownership
+
+The main purpose of this file is to make sure that someone is put in charge of the project and to prevent it from being forgotten. Another reason is to provide a contact for potential contributors. When no one other than the original author has maintenance rights, continuation of the project through bifurcation can lead to confusion about the status and reliability of the project.
+
+#### Test Directory
+
+Having tests on your project can avoid regresion, breaks and many other issues. A few reasons why testing is important: when bugs are caught in the earlier stages it costs much less to fix them; you add security; helps assure quality and digital confidence; which leads to users satisfaction. If your project has an automated test set, it will help to have changes incorporated into your project more quickly, easing the burden on reviewers and maintainers.
+
+#### Code of Conduct
+
+A code of conduct establishes expectations and facilitates a healthy and constructive community.
+
+_When a project seems hostile or unwelcoming, even if it&#39;s just one person whose behavior is tolerated by others, you risk losing many more contributors, some of whom you may never even meet. It&#39;s not always easy to adopt or enforce a code of conduct, but fostering a welcoming environment will help your community grow._
+
+You can read more about it at [Open Source Guide](https://opensource.guide/code-of-conduct/).
+
+#### LicenseOnReadme
+
+Although the license is valid in any format, as long as it is added to the project, having it in the main file ensures it will be clearly visible. So even if the license exists in a separate file it is important to have a mention of it in the README. Some projects add copyright and license on the first line of every file of the project.
+
+#### Security
+
+A SECURITY.md is where a security policy goes, a place to instruct users how to report a security vulnerability. This allows users to easily report and maintainers to quickly take actions upon vulnerabilities.
+
+#### Support
+
+A SUPPORT.md file is used to describe how the project is being maintained and how responsible a maintainer will be on issues. It can include: how and where a user can get help and support, which communication channels should be used and what not to do to avoid communication noise. This file helps both users and maintainers to communicate to each other having their needs to get into the right channel.
+
+Now imagine a project with all that. Will it help to build and keep a community? Will noise in communication be mitigated? Will it save tons of maintainers time to help people onboard and solve issues?
+
+Have I convinced you?
+
+So get a check-my-repo to call &quot;my own&quot; and make your friends envy you for how beautiful and healthy your repositories are.
```

Comment:
heh, okay, but maybe again we need to be clear that check-my-repo will only show you if there's some obvious deficiency.  to get to that beautiful repo, you need to put in some work!